README.md

```diff
@@ -29,11 +29,13 @@ cf restage $YOUR_APP_NAME
 
 #### Log Collection
 
-To start collecting logs from your application in CloudFoundry, the puppy-agent contained in the buildpack needs to be run and log collection enabled. Note that DogStatsD is automatically enabled with the puppy-agent.
+To start collecting logs from your application in CloudFoundry, the agent contained in the buildpack needs to be activated and log collection enabled.
 
 ```
 cf set-env $YOUR_APP_NAME RUN_PUPPY true
 cf set-env $YOUR_APP_NAME DD_LOGS_ENABLED true
+# Activate log collection only on the enabled agent
```

Comment:
I'd rather say something like `disable the core checks to disable system metrics collection`.